firehose.go

```diff
@@ -286,6 +293,7 @@ func (fp *Firehose) RunLeader(ctx context.Context) {
 			}
 			wg.Wait()
 			// restart scrape interval and run leader again
```

Comment:
comment needs to move one line below?

Reply:
I sort of considered both resetting current time and the timer channel as "restarting the scrape interval"

Reply:
because currentttime needs to be reset too otherwise the file is just overwritten

Reply:
makes sense.